core/scripts/compare-runs.js

```diff
@@ -143,8 +143,15 @@ async function gather() {
   const outputDir = dir(argv.name);
   if (fs.existsSync(outputDir)) {
     console.log('Collection already started - resuming.');
+  } else {
+    await mkdir(outputDir, {recursive: true});
+    fs.writeFileSync(`${outputDir}/meta.json`, JSON.stringify({
+      name: argv.name,
+      lhFlags: argv.lhFlags.split(' '),
+      urls: argv.urls,
+      n: argv.n,
```

Comment:
this is just nice to have for later reference, when looking inside `timings-data/`.